client/state/themes/full-site-editing.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { includes } from 'lodash';
+
+const FULL_SITE_EDITING_THEMES = [ 'maywood' ];
```

Comment:
Non-blocking: I'm kind of worried that these whitelists are spreading in different areas of the client, and that we'll end up having to update them in multiple places. I still have to audit all of the similar usages, but it would be good to keep this in mind going forward, and hopefully circle back and centralize them.

Reply:
I tried to centralize these checks as much as possible, but I definitely think we need to figure out how to handle theme support more generically.

We could perhaps add an override to this function which lets us pass in a theme, and expose that through the API:

https://github.com/Automattic/wp-calypso/blob/5b12ef3b9b8e79c19f87dc0d07072197e87150fc/apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php#L120-L122